browser_action/index.js

```diff
@@ -44,19 +58,24 @@
   });
 
   browser.storage.onChanged.addListener(async (change, areaName) => {
-    if (areaName !== "sync") return;
-
-    if (change.locations) {
-      $locations.innerHTML = "";
```

Comment:
This should be kept isn't it?

Reply:
No, I've moved the clear in the display method